### PR TITLE
Wall room

### PR DESCRIPTION
**Pull request by heinlson**

### CODE REVIEW

File: game/core/src/main/java/com/gg/game/model/Wall.java [hunk 1]
```diff
@@ -1,20 +1,46 @@
...
 
+import com.gg.game.model.math.Rectangle;
 import com.gg.game.model.utils.Point;
 
 /**
  * Represents the outer boundary of a room.
  */
-class Wall {
+class Wall implements Collidable {
+    private static final int Default_Wall_Size = 50;
```
Comment (pudkipz) [inline game/core/src/main/java/com/gg/game/model/Wall.java]: Not super important, but if we're following Google's style guide, constant names should use CONSTANT_CASE.

File: game/core/src/main/java/com/gg/game/model/Room.java [hunk 2]
```diff
@@ -7,7 +7,11 @@
 public class Room {
     private List<Enemy> enemies;
     private List<Decor> decor;
+    private static List<Wall> boundaries = initRoomWalls();
 
+    //TODO: move to constants file.
+    private static final int World_Width = 640;
+    private static final int World_Height = 460;
```
Comment (pudkipz) [inline game/core/src/main/java/com/gg/game/model/Room.java]: I recall the world height being 480, but I'm misunderstanding the use for World_Height?

Comment (heinlson) [reply game/core/src/main/java/com/gg/game/model/Room.java]: Will have to ask Keivan, but yes 480 sounds familiar

File: game/core/src/main/java/com/gg/game/model/Wall.java (hunk 1 above)
Comment (heinlson) [reply game/core/src/main/java/com/gg/game/model/Wall.java]: fixed

**Pull request merged**